.rubocop.yml

```diff
@@ -28,6 +28,16 @@ Metrics/LineLength:
   Max: 100
   Exclude:
     - Gemfile
-
+  
 Style/Documentation:
   Enabled: false
+
+
+Style/FormatString:
+  Enabled: false
+Style/DotPosition:
+  Enabled: false
+Style/TrailingWhitespace:
+  Enabled: false
```

Comment:
They're things _you_ consider to be non-issues... but for example _I_ certainly consider trailing whitespace to be a considerable issue. Dot position I don't hugely care about which choice we make, but I do prefer consistency to inconsistency.

Look at it this way: What if everybody disabled the couple of checks _they_ considered non-issues?